CHANGELOG.md

```diff
@@ -3,7 +3,9 @@
     * Made `ipy` consistent with `py` in the following ways
         * `ipy` returns whether any of the commands run in it returned True to stop command loop
         * `Cmd.in_pyscript()` returns True while in `ipy`.
-        * Starting `ipy` when `Cmd.in_pyscript()` is already True is not allowed.  
+        * Starting `ipy` when `Cmd.in_pyscript()` is already True is not allowed.
```

Comment:
Was this `ipy` change made in a different PR?  Because I don't see it in this one.  Other than that it looks OK.

Reply:
Yes, it's from a PR already merged into master. I think I changed white space on the line in this PR.